BuildAndTest.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
```

Comment:
there is also a file "UpdateBaselines.sh" which was for Linux only, not sure if we want to update that.

Reply:
What would be the proposed change? I don't see any reference in that file to BinSkimLinux.sln or the uname variable.

Reply:
oh because I see this file is running under folder /netcoreapp3.1/linux-x64/
I don't think we need another one running for macOS.